solutions/community-deploy/communitybuild.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+usage() { 
+echo "Generate cortx-all docker image from provided CORTX release build"
+echo "Usage: $0 [ -b brunch ] [ -p push docker-image to GHCR yes/no. Default no] [ -t tag latest yes/no. Default no" ] [ -e environment ] [ -h print help message ] 1>&2; exit 1; }
+while getopts "b:h:" opt; do
+    case $opt in
+        b ) BRANCH=$OPTARG;;
+        h ) usage
+        exit 0;;
+        *) usage
+        exit 1;;
+    esac
+done
+
+function DockerCheck() {
+        docker --version >/dev/null 2>&1
+        if [ $? -eq 0 ]; then
+                echo "Docker is installed and $(docker --version)."
+        else
+                echo "Docker not installed"
+                printf "\n################################################################################\n"
+                printf "\tInstalling Docker Engine\n"
+                printf "\n################################################################################\n"
+                yum install -y yum-utils
+                yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo -y
+                yum install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+                systemctl start docker
+                docker --version
+#               exit 1
+        fi
+}
+DockerCheck
+
+function DockerComposeCheck() {
+        docker-compose --version >/dev/null 2>&1
+        if [ $? -eq 0 ]; then
+                echo "Docker-Compose is installed and $(docker-compose --version)."
+        else
+                echo "Docker-Compose not installed"
```

Comment:
Source `functions.sh` and use `add_primary_separator`, `add_secondary_separator` or `add_common_separator` Refer - https://github.com/Seagate/cortx-re/blob/main/solutions/kubernetes/functions.sh#L21

Reply:
Done